### PR TITLE
Update win10 worker image to shared image

### PR DESCRIPTION
**Pull request by jwmoss**

### CODE REVIEW

**Review by jcristau — COMMENTED** [review]
Maybe clarify in the commit message that this is about the win10 "alpha" image?

File: worker-images.yml [hunk 1]
```diff
@@ -189,36 +189,20 @@ ronin-b3-windows2022-prod:
...
     west-us-2: trusted-win2022-64-2009-westus2-2022-datacenter-azure-edition
 # Windows 10
-ronin-t-windows10-64-2009-prod:
+ronin_t_windows10_64_2009_prod:
   azure2:
-    version: NA
-    resource_group: rg-packer-through-cib
+    version: 1.0.0
+    resource_group: rg-packer-worker-images
     deployment_id: "8346140"
```
Comment (jcristau) [inline worker-images.yml:196]: Just double checking that the deployment_id shouldn't change here?

**Review by jcristau — APPROVED** [review]
Is it worth running `/taskcluster integration` here to run a couple of tasks on stage?

**Comment by ahal** [comment]
Oops, Julien found a bug in my optimization, that's why it didn't schedule anything. Working on a fix.

**Comment by jcristau** [comment]
#232 should fix the optimization.

**Review by markcor — APPROVED** [review]

**Pull request merged**